_pages/positions.md

```diff
@@ -35,12 +35,16 @@ Please see the following for details of each application category.
 
 - [ ] &nbsp; Please submit your application through [LTI](https://www.lti.cs.cmu.edu/apply-lti). Unfortunately, we may not respond to a direct email about the Ph.D. application.
 
-### CMU undergraduate/master students
-
-- [ ] &nbsp; Please email your CV to shinjiw@ieee.org with the subject **"WAVLab undergraduate/master applications"**
-- [ ] &nbsp; If you do not put it in the subject, we regard that you do not thoroughly investigate our lab's activities in this webpage, and unfortunately, we may not respond to this email.
+### WAVLab Collaborator
+- We provide training and research opportunities to students who are interested in speech processing and want to collaborate with us.
+  - We give priority to students at Carnegie Mellon University (CMU); students from other universities are also welcome. We accept both undergraduate and graduate students.
+  - The opportunities are mainly educational-oriented, especially for students who are new to speech research. We guide students in understanding speech research background, required programming skills, knowledge of the cluster, and use of speech and audio toolkits. Students can also conduct research under our supervision. 
+  - It usually takes a few months or even longer for students to obtain sufficient research skills before starting serious research activities. We believe this process is necessary and is essential for new researchers. If you want to publish a paper as soon as possible, our lab is not the best option for you, unfortunately. We recommend you contact the other faculties.
+- [ ] &nbsp; Students who are interested in these opportunities should: 
+  - Fill out **this form**
```

Comment:
my understanding is that (please correct me if not true): there will be a form link attached here in later commits?

Reply:
Created first draft of Google form here: https://docs.google.com/forms/d/1AE-MMtqXpdPG07U6nnNfp9N2abJrI54XeVIshkyrwng/